testing.py

```diff
@@ -1084,7 +1084,10 @@ class ExternalSearchTest(DatabaseTest):
     to ensure that it works well overall, with a realistic index.
     """
 
+    SIMPLIFIED_TEST_ELASTICSEARCH = os.environ.get('SIMPLIFIED_TEST_ELASTICSEARCH', u'http://localhost:9200')
```

Comment:
It'll be nice to have this. I just ran into the non-default ES port issue last night while running tests with the docker containers for only one of the hosts of a Shared ODL dev environment. The ES host was on port 9210. Adding this to my env files now in anticipation.